src/components/ProfileForm/styles.js

```diff
@@ -0,0 +1,12 @@
+import { StyleSheet } from 'react-native';
+
+const styles = StyleSheet.create({
+  container: {
+    width: 240,
```

Comment:
I think this form it's not center on the screen right?

Reply:
Actually, it is, because the parent has an alignItems center wrapper. But, anyway, this doesn't look the same either in the current file.